resources/qml/Cura.qml

```diff
@@ -10,47 +10,35 @@ import QtQuick.Dialogs 1.2
 import UM 1.3 as UM
 import Cura 1.1 as Cura
 
+import "Dialogs"
 import "Menus"
+import "MainWindow"
 
 UM.MainWindow
 {
     id: base
-    //: Cura application window title
-    title: catalog.i18nc("@title:window","Ultimaker Cura");
-    viewportRect: Qt.rect(0, 0, (base.width - sidebar.width) / base.width, 1.0)
-    property bool showPrintMonitor: false
 
+    // Cura application window title
+    title: catalog.i18nc("@title:window", "Ultimaker Cura")
+    viewportRect: Qt.rect(0, 0, 1.0, 1.0)
```

Comment:
This is unnecessary, because it's already equal to the default.
```suggestion

```

Reply:
Done in 1cc7e0e586604ddf16b2931e1bcffbadb53c7f80